### PR TITLE
Update AddingExecutionProvider.md

### PR DESCRIPTION
**Pull request by migueldeicaza**
**Description**: Slight updates to guide folks trying to add a new execution provider for the first time

**Motivation and Context**

Spent a bunch of time googling to find answers with no luck, until I ran into a CI script,
so I am trying to solve the pain for future folks

### CODE REVIEW

**Comment by edgchen1** [comment]
FYI, this doc is moving here: https://github.com/microsoft/onnxruntime/blob/gh-pages/docs/how-to/add-execution-provider.md
See https://github.com/microsoft/onnxruntime/pull/6225

File: docs/AddingExecutionProvider.md [hunk 1]
```diff
@@ -35,3 +35,17 @@ e.g.
   OrtReleaseObject(factory);
   OrtCreateSession(env, model_path, session_option, &session);
 ```
+
+# Testing your execution provider
+
+To ease the testing of your execution provider, you can add a new case for it to the `onnx_test_runner` command,
+do this by adding it to `onnxruntime/test/onnx/main.cc` file, following the pattern for other existing providers.
```
Comment (gwang-msft) [inline docs/AddingExecutionProvider.md:42]: The onnxruntime_perf_test can be updated as well, to help measure the perf of new EP
onnxruntime/test/perftest/command_args_parser.cc

**Comment by migueldeicaza** [comment]
Well, looks like this will get automatically closed.   I feel bad for future folks that might want to avoid this pain.

**Comment by faxu** [comment]
@migueldeicaza, thank you so much for this contribution and apologies that this fell through the cracks. I've incorporated the changes in the latest iteration of that doc #8612, which is now hosted on the github.io documentation site.

**Pull request closed without merging**